package.json

```diff
@@ -9,7 +9,9 @@
   "scripts": {
     "prepare": "is-ci || husky install",
     "update:sdk": "yarn up @plentymarkets/shop-api",
-    "setup": "yarn && yarn build && yarn dev",
+    "setup": "if [ \"$OS\" = \"Windows_NT\" ]; then yarn setup:windows; else yarn setup:unix; fi",
```

Comment:
setup does not work if there is a .yarnrc.yml file with a different yarn version. We should use either use the force flag or have a some other information how to fix that.

Reply:
So I had to run ./yarn.sh -f to install update yarn.
Once that completed successfully, I ran into this issue when running the `yarn setup` again
<img width="1036" alt="Bildschirmfoto 2024-12-04 um 10 49 17" src="https://github.com/user-attachments/assets/686a4f57-c77b-4084-acdf-eb749b6c928e">
I guess after running yarn.sh i dont need the setup step. Still the setup command fails for me. Not sure if its becuase of zsh or something.

Reply:
and yes I did add the NPM_AUTH_TOKEN to the .env after running the yarn.sh and did run it again with no issues. just the yarn setup fails for me

Reply:
I can't replicate your issue here. For me it runs fine every time. Also I think we shouldn't use the force flag here, the command should prompt you for confirmation instead.

I've made one small change, can you try again?

Reply:
➜  plentyshop-pwa git:(chore/yarn-version) yarn setup
Unbound variable "OS"
command not found: then
command not found: else
command not found: fi

Reply:
What if you run `yarn setup:unix`? Does the actual command work as expected?